### PR TITLE
Adding new samples s390x/s390-unknown-linux-gnu

### PR DESCRIPTION
**Pull request by pradghos**
- s390x/s390 is allowed to pair with other vendor name.
- new samples are being added to use this flexibility.

Signed-off-by: Pradipta Ghosh pradghos@in.ibm.com

### CODE REVIEW

File: scripts/config.sub [hunk 1]
```diff
@@ -1238,6 +1230,7 @@ case $cpu-$vendor in
 			| sparclite \
 			| sparcv8 | sparcv9 | sparcv9b | sparcv9v | sv1 | sx* \
 			| spu \
+                        | s390 | s390x \
```
[the remark this answers is not in the document]
Comment (pradghos) [reply scripts/config.sub]: @cpackham : Fixed now. Thanks for looking.

**Comment by pradghos** [comment]
@cpackham : Please let me know if you have further review comments. Thank you !

**Comment by cpackham** [comment]
Could you squash the 2 changes together and push -f to update this PR so it's just one commit. I'd probably do it before merging anyway but if you do it on your end it'll save me some hassle.

**Comment by pradghos** [comment]
> 
> 
> Could you squash the 2 changes together and push -f to update this PR so it's just one commit. I'd probably do it before merging anyway but if you do it on your end it'll save me some hassle.

@cpackham I have squashed into one commit. Thanks!

**Pull request merged**